__mocks__/db/tasks.ts

```diff
@@ -1,248 +1,312 @@
 import task from '@/interfaces/task.type';
 
 const TASK = {
-    id: 'firestoreDocumentId123',
-    lossRate: {
-        dinero: 10,
-        neelam: 5,
-    },
-    links: ['https://realdevsquad.com/learn-site'],
-    completionAward: {
-        dinero: 110,
-        neelam: 10,
-    },
-    dependsOn: [],
-    assignee: 'shmbajaj',
-    startedOn: '1618790400',
-    isNoteworthy: true,
-    title: 'Testing and Determinsitic State',
-    purpose: 'string',
-    percentCompleted: 0,
-    endsOn: '1618790400',
-    status: 'progress',
-    featureUrl: 'progress',
-    type: 'feature',
-    createdBy: 'shmbajaj',
```

Comment:
keep the file format as original, it is hard to review the changes (if any) in
`__mocks__/db/tasks.ts`

Reply:
The "prettier:fix" command rule does not seem to be applied to the mock database files. As a result, I had to manually fix the formatting of the data.

Reply:
Please note that this is just dummy data, and there are no logic or code issues that would affect the application.

Reply:
You can skip this file.

Reply:
Test is passing right?